src/adaptix/_internal/feature_requirement.py

```diff
@@ -163,6 +163,7 @@ def fail_reason(self) -> str:
 HAS_TYPED_DICT_REQUIRED = HAS_PY_311
 HAS_SELF_TYPE = HAS_PY_311
 HAS_TV_TUPLE = HAS_PY_311
+HAS_UNPACK = HAS_PY_311
```

Comment:
There are still other cases of usage ``HAS_TV_TUPLE`` that require ``typing.Unpack``. It has to be replaced with HAS_UNPACK